google/cloud/firestore_v1/query.py

```diff
@@ -203,7 +223,7 @@ def _chunkify(
             snapshots = _q.get()
 
             if snapshots:
-                last_document = snapshots[-1]
+                last_document = snapshots[-1]  # type: ignore
```

Comment:
why type ignore?

Reply:
Some unit test wasn't passing but somehow it works now. I'll undo the change.